docker/solr/2.10.Dockerfile

```diff
@@ -1,4 +1,58 @@
-FROM ckan/ckan-solr:2.10-solr8-spatial
+FROM --platform=$TARGETPLATFORM solr:8
```

Comment:
I'll have to test it out locally to see if this still works with CKAN 2.10.4, and then we will probably have to update the patch version for solr in the build image GHA